libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -1,18 +1,37 @@
+
+// tslint:disable-next-line: nx-enforce-module-boundaries
+import { addToReadingList } from './../../../../data-access/src/lib/+state/reading-list.actions';
```

Comment:
You want to take advantage of the path alias for your library, as specified here: https://github.com/VenkataBodempudi/web-ui-developer-puzzle/blob/master/tsconfig.base.json#L25
It's actually already included in your file. This way you don't have to include these long relative paths, or the tslint exclude.